sandbox/cgroup/scanning.go

```diff
@@ -50,20 +57,39 @@ func securityTagFromCgroupPath(path string) naming.SecurityTag {
 	// from a snap.
 	// Expected format of leaf name:
 	//   snap.<pkg>.<app>.service - assigned by systemd for services
-	//   snap.<pkg>.<app>.<uuid>.scope - transient scope for apps
-	//   snap.<pkg>.hook.<app>.<uuid>.scope - transient scope for hooks
+	//   snap.<pkg>.<app>-<uuid>.scope - transient scope for apps
+	//   snap.<pkg>.hook.<app>-<uuid>.scope - transient scope for hooks
 	if ext := filepath.Ext(leaf); ext != ".service" && ext != ".scope" {
 		return nil
 	}
 
-	// There are two broad forms expressed by the pair of regular expressions defined above.
+	// The original naming convention for scope transient units was
+	// snap.<pkg>.<app>.<uuid>.scope, but is now
+	// snap.<pkg>.<app>-<uuid>.scope.
+	//
+	// Check for the new patterns first, and then fall back to the old
```

Comment:
old is a bit strange as they cover also services as the comment clarifies, maybe s/old/original/

Reply:
Fixed, thanks!